### PR TITLE
Improved an amazing project

### PR DESCRIPTION
**Pull request by ssisodiya28**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -64,8 +62,7 @@ list contains your navigation items, ready to output like in the examples above.
 
 Caching:
 --------
-To avoid hitting the database every time a user requests a page, the menu items are 
-cached if you have a cache configured. Caching is not used when ``settings.DEBUG`` is ``True``.
+To avoid hitting the database every time a user requests a page, the menu items are cached if you have a cache configured. Caching is not used when ``settings.DEBUG`` is ``True``.
```
Comment (ollytheninja) [inline README.rst:65]: As above keeping line lengths short is a valid choice by the author and should stay as such unless there is a good reason for it.

File: README.rst [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
+maintained by the staff using the Django administration without any knowledge of HTML or Django internals.
 
-Sub-menus can also be easily built and displayed only for particular URIs.
+The Sub-menus can also be easily built and displayed only for particular URIs.
 
 Installation & Configuration:
 -----------------------------
 
-1. ``pip install django-menu``
+1. ``pip install Django-menu``
```
Comment (ollytheninja) [inline README.rst:14]: Backticks (\`) are used for syntax highlighting in markdown - it generally indicates that the text in the markdown  refers to code or a typed command where capitalisation and spaces matter
In this case the command is different with an upper case `D` in `Django`
https://guides.github.com/features/mastering-markdown/

File: README.rst [hunk 3]
```diff
@@ -40,7 +38,7 @@ Installation & Configuration:
                    },
                ]
 
-5. Add a Menu (eg called ``headernavigation``) and add some items to that menu
+5. Add a Menu (eg called ``header navigation``) and add some items to that menu
```
Comment (ollytheninja) [inline README.rst:41]: as above this changes the meaning of the word within the backticks

File: README.rst [hunk 4]
```diff
@@ -1,25 +1,23 @@
...
 -----------
 
 BSD-licensed menu tools for Django, built by Ross Poulton <http://www.rossp.org>
 
-django-menu provides a basic structure for you to build multiple navigation 
-menus for your website, such as the header menubar. These menus can be easily 
-maintained by staff using the Django administration without any knowledge 
-of HTML or Django internals.
+Django-menu provides a basic structure for you to build multiple navigation menus for your websites, like the header menubar. These menus can be easily 
+maintained by the staff using the Django administration without any knowledge of HTML or Django internals.
```
Comment (ollytheninja) [inline README.rst:7]: It looks like the author chose to stay between 80 or 120 characters per line; this is supported by Markdown, is not reflected in the rendered text and is a valid choice.
Keeping line lengths short increases readability in text editors and when viewing text files in the terminal.
Consider keeping this section in multiple, shorter lines

**Review by ollytheninja — CHANGES_REQUESTED** [review]
As per comment inline, the majority of the changes in this PR do not improve the document, some changes are incorrect.

**Comment by rossp** [comment]
@ssisodiya28  Sorry, this doesn’t improve the project. Submitting what appears to be an automated pull request from a code linter, that makes the documentation inaccurate, wastes the time of project maintainers. Please do not do this. 

@ollytheninja thank you for taking the time to review this so quickly.

**Comment by GGG-KILLER** [comment]
@rossp this is probably because of Hacktoberfest, mark this request with the `invalid` or `spam` tag to make it not count towards their quota as non-tagged closed PRs still do.

**Comment by rossp** [comment]
Thanks for pointing that out. I'd tried answering in good faith but obviously spent more time doing that than they did on their submissions. I get the idea behind Hacktoberfest but on an old, small, specific-use-case project it's a waste of time.

I'll tag those items as spam. Thanks again.

**Pull request closed without merging**